src/fundus/publishers/us/techcrunch.py

```diff
@@ -14,10 +14,42 @@
 
 
 class TechCrunchParser(ParserProxy):
+    class V2(BaseParser):
+        _summary_selector: XPath = CSSSelector("div.entry-content > p#speakable-summary")
+        _paragraph_selector: XPath = CSSSelector("div.entry-content > p:not(#speakable-summary)")
```

Comment:
Do we want to also parse bulletpoint lists? If so, I have found one usage: https://techcrunch.com/2024/05/15/senate-study-proposes-at-least-32b-yearly-for-ai-programs/